tests/e2e/appsettings.user.json

```diff
@@ -1,6 +1,6 @@
 {
-  "Forge": {
-    "ClientId": "<your client id>",
-    "ClientSecret": "<your client secret>"
-  }
+    "Forge": {
+        "ClientId": "CokbOqRvp41DVunD7lBO2ciiwJpltTAf",
+        "ClientSecret": "68d7Go7R9QtGDR9Z"
+    }
```

Comment:
should we remove these credentials?

Reply:
Yes. I didn't mean to add these. Rotated the secret...